### ISSUE TITLE
Working with Events 

### ISSUE THREAD

**Issue by efkan** [socketio/socket.io-client-java#331]
Hi,

**tl; dr ,** I thought that I can listen the all events with their `args` below. Can not I?

To learn how to use `socket.io-client-java`, I created a method and put into it the all events which placed in the [Socket.java](https://github.com/socketio/socket.io-client-java/blob/master/src/main/java/io/socket/client/Socket.java#L66-L78) file like the below:

	public void connect () {
		mSocket.on(Socket.EVENT_CONNECT, onConnect);
		mSocket.on(Socket.EVENT_CONNECTING, onConnecting);
		mSocket.on(Socket.EVENT_DISCONNECT, onDisconnect);
		mSocket.on(Socket.EVENT_ERROR, onError);
		mSocket.on(Socket.EVENT_CONNECT_ERROR, onConnectError);
		mSocket.on(Socket.EVENT_CONNECT_TIMEOUT, onConnectTimeout);
		mSocket.on(Socket.EVENT_RECONNECT, onReconnect);
		mSocket.on(Socket.EVENT_RECONNECTING, onReconnecting);
		mSocket.on(Socket.EVENT_RECONNECT_ERROR, onReconnectError);
		mSocket.on(Socket.EVENT_RECONNECT_FAILED, onReconnectFailed);
		mSocket.on(Socket.EVENT_RECONNECT_ATTEMPT, onReconnectAttempt);
		mSocket.on(Socket.EVENT_PING, onPing);
		mSocket.on(Socket.EVENT_PONG, onPong);

		// Can use a custom channel here!
		// mSocket.on("new message", onNewMessage);

		mSocket.connect();
	}

My onConnecting listener. The exception specifies `mActivity.runOnUiThread` line.

	private Emitter.Listener onConnecting = new Emitter.Listener() {
		@Override
		public void call(final Object... args) {
			mActivity.runOnUiThread(new Runnable() {
				@Override
				public void run() {

				}
			});
		}
	};

Then I wrote Emitter.Listener methods for the all events. However, my app throws an exception like the following:

```
E/AndroidRuntime( 3881): FATAL EXCEPTION: EventThread
E/AndroidRuntime( 3881): java.lang.NullPointerException
```

**Comment by efkan** [comment]
False alarm!..
